monitoring/status-page/status/routes.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import json
+import requests
+from flask import render_template
+from status import app
+
+watchdog = 'http://watchdog.hmny.io/status-%s'
```

Comment:
do you need to hardcode the URL?  consider to use reverse proxy by nginx as well, like we did for watchdog.

Reply:
The URL is for getting the network status data from existing Watchdog services, not for serving the webpage.